sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -1460,4 +1460,29 @@ class AdaptiveQueryExecSuite
       }
     }
   }
+
+  test("SPARK-33933: Materialize BroadcastQueryStage first in AQE") {
+    val testAppender = new LogAppender("aqe query stage materialization order test")
+    val df = spark.sparkContext.parallelize(Range(0, 10))
```

Comment:
seems we can simplify it: `val df = spark.range(1000).select($"id" % 26, $"id" % 10).toDF("index", "pv")`

Reply:
sure, since no sleep is needed